core/src/main/scala/org/typelevel/paiges/Chunk.scala

```diff
@@ -66,8 +74,17 @@ private[paiges] object Chunk {
       case (i, Doc.Concat(a, b)) :: z => loop(pos, (i, a) :: (i, b) :: z)
       case (i, Doc.Nest(j, d)) :: z => loop(pos, ((i + j), d) :: z)
       case (_, Doc.Align(d)) :: z => loop(pos, (pos, d) :: z)
-      case (i, Doc.Text(s)) :: z => ChunkStream.Item(s, pos + s.length, z, false)
-      case (i, Doc.Line(_)) :: z => ChunkStream.Item(null, i, z, true)
+      case (i, Doc.Text(s)) :: z => ChunkStream.Item(s, pos + s.length, None, z, false)
+      case (i, Doc.Line(_)) :: z =>
+        if (!trim) ChunkStream.Item(null, i, None, z, true) else {
```

Comment:
I'd rather both the if and the else have `{` or neither have them. If you don't agree feel free to leave it.